src/crate/theme/rtd/crate/sections/footer.html

```diff
@@ -5,7 +5,124 @@
         <div class="row">
           <div class="col-md-4 text-md-center">
             <a class="w-inline-block" href="/" title="cratedb.com">
-              <img src="{{ pathto('_static', 1) }}/images/cratedb-logo.svg" alt="CrateDB Logo" width="218" height="36" loading="lazy">
+              <svg version="1.2" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 218 36" width="218" height="36" alt="CrateDB Logo" loading="lazy" class="cr-cratedb-logo">
+                <defs>
+                  <clipPath clipPathUnits="userSpaceOnUse" id="cp1">
+                    <path d="m1.49 3.63h7.31v7.31h-7.31z" />
```

Comment:
Is it a good idea to inline that so many times here? Also redundantly in `header-content.html` and `header.html`?